.circleci/config.yml

```diff
@@ -48,52 +66,56 @@ jobs:
           command: rake dependencies
       - run:
           name: Build for Testing
-          command: cd ./Scripts && bundle exec fastlane build_for_testing
+          command: cd ./Scripts && bundle exec fastlane build_for_testing device:'<< parameters.device >>' ios-version:'<< parameters.ios-version >>'
```

Comment:
That's an interesting fix that I missed the PR for; was this applied to other iOS repos as well?